docs/self-managed/zeebe-deployment/zeebe-gateway/job-streaming.md

```diff
@@ -0,0 +1,23 @@
+---
+id: job-streaming
+title: "Job streaming"
+sidebar_label: "Job streaming"
+description: "Streaming job workers is expected to be long-lived to cut down on the latency overhead involved with re-creating a stream and propagating this throughout the cluster."
+---
+
+Streaming job workers is expected to be long-lived to cut down on the latency overhead involved with re-creating a stream and propagating this throughout the cluster. This may require special configuration, especially if you're using a reverse proxy in front of your gateway. Typically, this will affect you in the form of HTTP 504 (gateway timeout) being returned to your job streaming worker at regular intervals.
```

Comment:
What do we mean by "Streaming job workers is expected to be long-lived" -- as in, we expect to support this longterm?

Reply:
No, it's that job workers are expected to _run_ for a long time, i.e. not just connect once for a single job and then disappear again. It's better when they are long-lived because the initial setup, i.e. establishing connections, registering it with the gateways and brokers etc. is relatively expensive and shouldn't be done very often.